litmus-portal/frontend/src/views/Analytics/Overview/index.tsx

```diff
@@ -227,12 +221,13 @@ const Overview: React.FC = () => {
           link={
             <Link
               underline="none"
-              color="primary"
               href="https://prometheus.io/docs/introduction/overview/"
               target="_blank"
               rel="noreferrer"
             >
-              <Typography>Read prometheus doc</Typography>
+              <TextButton variant="highlight">
+                <Typography>Read prometheus doc</Typography>
```

Comment:
Can this be added in translation file?

Reply:
All the translations for analytics will be raised as a separate PR as per earlier discussion @amityt